test/system/030-run.bats

```diff
@@ -305,4 +305,22 @@ echo $rand        |   0 | $rand
     run_podman wait $cid
 }
 
+@test "podman run --log-driver" {
+    run_podman run --log-driver k8s-file --log-opt path=/var/log/podmantest.log --rm $IMAGE echo "podman log-driver test"
+    run grep "podman log-driver test" /var/log/podmantest.log
```

Comment:
Could you use $PODMAN_TMPDIR for this tests to make sure we don't effect the system, or cause problems with multiple tests running at the same time.